sdk/__tests__/port/out/DFNSTransactionAdapter.test.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ *
+ * Hedera Stablecoin SDK
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {
+	AssociateTokenRequest,
+	CreateRequest,
+	InitializationRequest,
+	Network,
+	RequestPublicKey,
+	StableCoin,
+	StableCoinViewModel,
+	SupportedWallets,
+	TokenSupplyType,
+} from '../../../src';
+import { MirrorNode } from '../../../src/domain/context/network/MirrorNode';
+import { JsonRpcRelay } from '../../../src/domain/context/network/JsonRpcRelay';
+import ConnectRequest, {
+	DFNSConfigRequest,
+} from '../../../src/port/in/request/ConnectRequest';
+import {
+	DFNS_SETTINGS,
+	FACTORY_ADDRESS,
+	HEDERA_TOKEN_MANAGER_ADDRESS,
+} from '../../config';
+import Injectable from '../../../src/core/Injectable';
+import * as fs from 'fs';
+import * as path from 'path';
+
+const decimals = 6;
+const initialSupply = 1000;
+const apiSecretKey = fs.readFileSync(
+	path.resolve(DFNS_SETTINGS.serviceAccountPrivateKeyPath),
+	'utf8',
+);
+
+describe('🧪 DFNSTransactionAdapter test', () => {
+	let stableCoinHTS: StableCoinViewModel;
+
+	const delay = async (seconds = 5): Promise<void> => {
+		seconds = seconds * 1000;
+		await new Promise((r) => setTimeout(r, seconds));
+	};
+
+	const mirrorNode: MirrorNode = {
```

Comment:
This can use the ENV related to the nodes instead of be hardcoded per test. They are in config.ts 😉

Reply:
changed